ITE/2/README.adoc

```diff
@@ -0,0 +1,473 @@
+makes a terminating delegation of all packages ("`*`") to the
+“package-and-in-toto-metadata-signer'' role. It MUST use a different set
+of (m, n) offline keys from the root role, where n >= 2, and m >=
+ceiling(n / 2), and its metadata SHOULD expire in 1 year. See `targets.json`
+for an example.
+
+.targets.json
+[source,json]
+----
+{
+  "signatures": {...},
+  "signed": {
+    ...,
+    "delegations": {
+      "keys": {...},
+      "roles": [
+        {
+          "keyids": [...],
+          "name": "packages-and-in-toto-metadata-signer",
+          "paths": [
+            "*"
+          ],
+          "terminating": true,
+          "threshold": 1
+        }
+      ]
+    },
+    "targets": {
+      "in-toto-metadata/root.layout": {
+        "hashes": {
+          "sha256": "930c48fa182d14835febd6a7f9129e34b83246f74238b9747fef7fc12147184d",
+          "sha512": "6fb781b534266411d0c424626b728b57e6c0a39b21798729efc63ff73556dfd19ebeddf7612da272936dad890d71b7e3caa65735ab6ac293740f2c5d29795590"
+        },
+        "length": 101047
+      },
+      "in-toto-pubkeys/298f37401f0b526a708967b7f708bc9c938fe0ad4bfe50d66837c20a57084e84.pub": {
+        "hashes": {
+          "sha256": "a19b11a130b35fb205e8cf8ab2f2488f387332be56857968785ce9899a521b05",
+          "sha512": "b7e48526cdde5c879e710c9542b4cc6b36de91ed93d709d2f5be4ec45d719e77397f69b6b8c117826987418d334eb01e9d27b37874c66157b536a77035b44048"
+        },
+        "length": 799
+      },
+      "in-toto-pubkeys/3e82bcdc71b29999340ceaadf3dc4193f8b06572d1c20612e9acdd7b52fa4b90.pub": {
+        "hashes": {
+          "sha256": "3560de9da223ac51b5cdbf25acf9f8e8f9f7b699eeda912c7a26a68c5f01ce12",
+          "sha512": "65aa1c2e7114714224142812bfc7bb8b2e6a37f54ebdbcbe20cc6e8f1504d73df17747a3ccc5acbdbefbe4df58065f88c72e4c4dac7e50c067c05b8b2ee25f4f"
+        },
+        "length": 799
+      },
+      "in-toto-pubkeys/e847f58ca5e83fc48d1d2388ddd8f1a168b205a3fe7978ad015dee3ae7b2ecf7.pub": {
+        "hashes": {
+          "sha256": "8cb4a254ae123a8bd91b1c9abdd99e719aa8349ff7eafd168988ce8a935d51a1",
+          "sha512": "e259f98b766537ed2893c1b1e25d171d8ab374702f29d0fbe3708b13a4456e153b29e36722f136bc963e4a85fa7581dfbbf40ebd3e1538227ec30874264ddd2b"
+        },
+        "length": 799
+      }
+    }
+    ...,
+
+  }
+}
+----
+
+The snapshot and timestamp roles SHOULD use (1, 1) online keys, and
+their metadata SHOULD expire in 1 day. The in-toto "`packager`"
+functionary, as well as the TUF timestamp, snapshot, and
+"`package-and-in-toto-metadata-signer`" roles, MAY share the same (1, 1)
+online keys. Furthermore, these online TUF roles MAY share the same
+metadata expiration period (e.g., 1 day). Online keys are used so that
+robots can use the key to sign new metadata on demand. When the in-toto
+or TUF implementation supports it, it is RECOMMENDED that online keys
+be protected from  exfiltration using a Hardware Security Module
+(HSM).
```

Comment:
What advantages does an HSM give?  Talking about the real world impact of this is helpful.